packages/@atjson/document/src/index.ts

```diff
@@ -231,16 +231,26 @@ export default class Document {
       return this.clone() as InstanceType<To>;
     // Coerce or convert to new type
     } else {
-
       class ConversionDocument extends DocumentClass {
         static schema = DocumentClass.schema.concat(to.schema);
       }
-      let convertedDoc = new ConversionDocument(this.toJSON());
+
+      let convertedDoc = new ConversionDocument({
+        content: this.content,
+        annotations: this.where({}).sort().annotations
```

Comment:
This has to be sorted; Idk, but prism fails when the annotations aren't sorted 🙃